pkg/controller/storagecluster/cephobjectstores.go

```diff
@@ -55,12 +55,16 @@ func (r *ReconcileStorageCluster) createCephObjectStores(cephObjectStores []*cep
 			cephObjectStore.ObjectMeta = existing.ObjectMeta
 			err = r.client.Update(context.TODO(), cephObjectStore)
 			if err != nil {
+				reqLogger.Error(err,
+					fmt.Sprintf("Failed to update CephObjectStore Object: %s", cephObjectStore.Name))
 				return err
 			}
 		case errors.IsNotFound(err):
 			reqLogger.Info(fmt.Sprintf("Creating CephObjectStore %s", cephObjectStore.Name))
 			err = r.client.Create(context.TODO(), cephObjectStore)
 			if err != nil {
+				reqLogger.Error(err,
+					fmt.Sprintf("Creation failed for CephObjectStore object: %s", cephObjectStore.Name))
```

Comment:
Move all changes to this file in this second commit to the first commit. These are part of the refactoring and do not impact the addition of the external CephObjectStore.

Reply:
Done...